internal/experiment/nodegroup.go

```diff
@@ -3,16 +3,18 @@ package experiment
 import (
 	"errors"
 
+	"github.com/stg-tud/bp2022_netlab/internal/customtypes"
 	"github.com/stg-tud/bp2022_netlab/internal/movementpatterns"
 )
 
 // A NodeGroup represents a group of dependent nodes sharing properties
 // such as a MovementModel or network settings.
 type NodeGroup struct {
-	Prefix  string
-	NoNodes uint
-
-	MovementModel movementpatterns.MovementPattern
+	Prefix             string
+	NoNodes            uint
+	PredefinedPosition bool
```

Comment:
Why do you use a combined type (active bool & payload data) here as well, as you did with the core automator pattern?